docs/metareview/context/mrv-20260827-180059866454000-task-done-process-overrides-93d185e3-context.md

```
@@ -0,0 +1,1133 @@
+- Risk level: `none`
+- Generated files excluded: docs/metareview/context/mrv-20260827-174858700250000-task-done-process-overrides-93d185e3-context.md, docs/metareview/reviews/mrv-20260827-174858700250000-task-done-process-overrides-93d185e3.md
+
+
+
+## Review Manifest
+
+- Manifest verdict: `NEEDS_REVISION`
+- Source manifest hash: `8a04de6f15d9595b`
+- Runtime assessment: static-only; runtime not assessed
+
+### Source Paths
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)